CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog
 
+## 2.5.5
+
+- Change Flow types to work with newer versions of Flow.
```

Comment:
Will this cause any new flow errors for users?

Reply:
I'm confident that it won't. @emmajam is on Flow 0.78.x and she put this code in manually and it didn't cause any errors.